app/Dockerfile

```diff
@@ -31,15 +31,27 @@ RUN groupadd -r runner && \
     chown -R runner:runner /home/runner/.ssh
 
 # Install script dependencies
-RUN apt-get update -y && apt-get upgrade -y
-RUN pip install --no-cache-dir -r /tmp/ADT/scripts/packages.txt
+RUN yum update -y
+RUN yum install -y wget make gcc openssl-devel zlib-devel pcre-devel bzip2-devel libffi-devel epel-release
+
+# Install Python3.7.2 and pip modules
+RUN cd /usr/bin && \
+    wget https://www.python.org/ftp/python/3.7.2/Python-3.7.2.tgz && \
+    tar xzf Python-3.7.2.tgz && \
+    cd Python-3.7.2 && \
+    ./configure --enable-optimizations && \
+    make altinstall && \
+    alternatives --install /usr/bin/python python /usr/local/bin/python3.7 1
+RUN python -m pip install --no-cache-dir -r /tmp/ADT/scripts/packages.txt
+
+# Fix yum installer with Python3.7 running as a global default
```

Comment:
lines 47:49
Comment references Python3.7, code references python2.7.

Reply:
Yes. That's correct yum needs to run under Python2.7